src/Modules/Payments/Domain/PriceListItems/PriceListItem.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using CompanyName.MyMeetings.BuildingBlocks.Domain;
+using CompanyName.MyMeetings.Modules.Payments.Domain.MeetingPayments;
+using CompanyName.MyMeetings.Modules.Payments.Domain.PriceListItems.Events;
+using CompanyName.MyMeetings.Modules.Payments.Domain.SeedWork;
+using CompanyName.MyMeetings.Modules.Payments.Domain.Subscriptions;
+
+namespace CompanyName.MyMeetings.Modules.Payments.Domain.PriceListItems
+{
+    public class PriceListItem : AggregateRoot
+    {
+        private string _countryCode;
+        
+        private SubscriptionPeriod _subscriptionPeriod;
+
+        private PriceListItemCategory _category;
+
+        private MoneyValue _price;
+
+        private bool _isActive;
+
+        private PriceListItem()
+        {
+        }
+
+        public static PriceListItem Create(
+            string countryCode,
+            SubscriptionPeriod subscriptionPeriod,
+            PriceListItemCategory category,
+            MoneyValue price)
+        {
+            var priceListItem = new PriceListItem();
+            
+            var priceListItemCreatedEvent = new PriceListItemCreatedDomainEvent(
+                Guid.NewGuid(), 
+                countryCode,
+                subscriptionPeriod.Code,
+                category.Code,
+                price.Value,
+                price.Currency,
+                isActive: false);
```

Comment:
In my opinion, it should be active by default.

Reply:
Ok, made it active. Actually i implemented it active by default at the begining. But then i looked again at commands in Miro and decided that create-change-activate-deacticate was a implying workflow.

Reply:
I agree, It was not clear